libsolidity/formal/SMTEncoder.cpp

```diff
@@ -861,6 +857,9 @@ void SMTEncoder::endVisit(Identifier const& _identifier)
 		defineExpr(_identifier, m_context.state().thisAddress());
 		m_uninterpretedTerms.insert(&_identifier);
 	}
+	// Ignore struct type identifiers in struct constructor calls
```

Comment:
In what order are identifiers and function calls visited? I'm assuming (going by this change), identifier first, and then function calls?

Reply:
Yes, FunctionCall is visited by endVisit, so your sequence is correct. I think only a few don't use endVisit, like MemberAccess